projects/forms/src/lib/forms-typed.ts

```diff
@@ -129,6 +129,7 @@ export interface TypedFormGroup<K, C extends Controls<K> = TypedControlsIn<K>> e
   value: K;
   setControl: <T extends keyof C>(name: T extends string ? T : never, control: C[T]) => void;
   reset: (value?: ResetValue<K>, options?: FormEventOptions) => void;
+  getRawValue: () => K;
```

Comment:
Great, I'd appreciate it if you added a test case and/or a use case respectively in `forms-typed.tests.ts` and in some of the components in the `src` example app.

Reply:
For the example app I was just gonna add a `console.log()` to show that the values from `getRawValue()` are different from `value` with disabled controls. Is that sufficient?

Reply:
Okay, I've never seen testing in the style you're using and I'm at a loss for how to do it in a way that would be satisfactory.